### PR TITLE
[14.0][MIG] stock_request_direction

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Comment by kittiu** [comment]
@Freni-OSI 

In test, I found error related to field `security_lead`, which is related to sale_stock. So, you need to make sure that this stock_request_direction and sale_stock do not run together.

I couldn't find the sample, but here was the suggesting from Stephane in the mailing list. I just copy it here,

```
In general you should avoid modifying repo-level dotfiles such as .travis.yml manually, so they can be kept in sync with the template automatically.

The correct way to test some modules independently from others is to run "copier update" and answer the rebel_module_groups question.
This will generate the correct Travis configuration and will be compatible with the future GitHub actions config too.

So, for example, if you answer ["a", "b,c"], it will create 3 groups to test
- a alone
- b and c only
- all the modules in the repo except a, b, c
```

**Comment by rousseldenis** [comment]
@Freni-OSI I think this is a [MIG] for ...  'migration' :smiley:

**Comment by rousseldenis** [comment]
@Freni-OSI Do you plan to finish this ? Could you squash not meaningful commits ? Thanks

**Comment by dreispt** [comment]
Rescuing this. Commits squashed.

File: stock_request_picking_type/tests/test_stock_request.py [hunk 1]
```diff
@@ -108,18 +108,29 @@ def test_onchanges_order(self):
             )
         )
 
-        form = Form(self.env["stock.request.order"])
```
Comment (rousseldenis) [inline stock_request_picking_type/tests/test_stock_request.py:111]: Why is the Form() behaviour is not convenient ?

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@Freni-OSI For further migrations, please follow this guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Review by dreispt — APPROVED** [review]

**Comment by AmmarOfficewalaSerpentCS** [comment]
@dreispt , 
can we help on anything merge this PR?
we planning  to migrate on a v15

**Comment by dreispt** [comment]
@AmmarOfficewalaSerpentCS Merge was already triggered but had build errors, that may need fixing I can retry just in case it might work now.

/ocabot merge nobump

**Comment by dreispt** [comment]
What do we need to do to allow the bot rebase work?

**Comment by dreispt** [comment]
@Freni-OSI Tests are failing. Maybe making the field required only at form level can make them pass.
https://app.travis-ci.com/github/OCA/stock-logistics-warehouse/jobs/564694118#L3269

**Comment by kittiu** [comment]
@Freni-OSI can you help fix/rebase please?

**Comment by dreispt** [comment]
Rebased.

**Comment by TheCloneMaster** [comment]
Travis is complaining that "direction" is a required field...

**Comment by Freni-OSI** [comment]
It's ready for review.
CC: @rousseldenis @dreispt

**Pull request merged**